deCODE_to_MT.py

```diff
@@ -90,8 +89,17 @@ def main(
         )
         | (hl.len(mt.alleles) != 2)
         | ((hl.len(mt.alleles) == 2) & (mt.n_unsplit_alleles != 2))
-        | (hl.is_missing(mt["InbreedingCoeff"]))
-        | ((hl.is_defined(mt["InbreedingCoeff"])) & (mt["InbreedingCoeff"] < -0.3))
+    )
+    mt = mt.filter_rows(filter_conditions, keep=False)
+
+    # Exclude variants with inbreeding coefficient < -0.3
+    # InbreedingCoeff was calculated by function 'bi_allelic_site_inbreeding_expr'
+    mt = hl.variant_qc(mt)
+    mt = mt.annotate_rows(InbreedingCoeff=bi_allelic_site_inbreeding_expr(mt.GT))
```

Comment:
Just noting that this will likely yield different results than running this computation _before_ filtering samples (line 61), so make sure this is the order you want.

Reply:
Thank you for checking Leo! I am just thinking this coeff should be more reliable if calculated after excluding samples with poor quality (e.g., mismatched sex, etc)? 
Another clue is that @vladsavelyev commented in the thread - "I think it makes sense re-calculate InbreedingCoeff in your downstream pipeline". I guess this should mean it is ok to calculate InbreedingCoeff after general QC?